project.go

```diff
@@ -51,7 +51,7 @@ func (p *Project) GetName() string {
 
 // Up brings up the specified project container with its dependencies.
 func (p *Project) Up(appConfig *c.Dev) {
-	RunComposeUp(appConfig.ImagePrefix, p.Config.DockerComposeFilenames, "-d")
+	RunComposeUp(appConfig.ImagePrefix, p.Config.DockerComposeFilenames, "-d", "--no-build")
```

Comment:
with `--no-build` flag, `dev <app> up` will no longer re-install the deps? if so we will make sure to announce this in the new release of `dev`.

Reply:
Yes. Right now it can lead to problems because if the base-bionic layer or the dev layer isn't built yet then it will just fail when someone runs up. Also if you mix and match docker-compose versions, it will actually rebuild the whole container from scratch, annoyingly. It would be better to just fail I think.